auror_core/v2/loader.py

```diff
@@ -0,0 +1,34 @@
+import os
+import copy
+
+import yaml
+
+from auror_core.v2.job import Job
+from auror_core.v2 import JobType
+
+
+class YAMLLoader:
```

Comment:
v2 flows are always YAML... Perhaps it's better to call this class only as "Loader" because it's already in v2 package

Reply:
Ok, it's a good point! If always will be an YAML file, we can short the name.